### PR TITLE
Add container metrics introduction post

### PR DESCRIPTION
**Pull request by chasballew**
@krallin for content
@sandersonet - for fixing the image overflows

### CODE REVIEW

File: introducing_container_metrics.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  2. Our container quickly consumed 100% of its 1 GB of available memory. Most of the memory was allocated for kernel page caches, which Linux uses to minimize expensive I/O requests.
+
+  3. With a load average consistently over 20 (i.e. > 20 tasks blocked waiting on CPU or I/O), our database operations are going to be very delayed. If our app was experiencing slowdowns around the same time, our database would be a likely suspect.
+
+Armed with that knowledge, what can we do? A high load average can be caused by a bottleneck in terms of I/O or CPU, or both. Detailed CPU and I/O metrics are coming soon. In the meantime, upgrading to a bigger container might help with both our problems:
+
+  * Our CPU allocation would be bigger, which essentially means we’d run CPU tasks faster.
+
+  * Our memory allocation would be bigger, which means more memory for caches and buffers, which means faster disk reads (disk writes on the other hand would probably not be faster, since it’s important that they actually hit the disk for durability, instead of sit in a buffer).
```
Comment (krallin) [inline introducing_container_metrics.md]: We should probably edit to "*rather* than sit in a buffer" (sorry I didn't catch that one earlier :/)

**Comment by sandersonet** [comment]
The `img` fix is merged on www

**Comment by krallin** [comment]
Left one comment, LGTM besides that :)

**Comment by chasballew** [comment]
Merging and releasing to staging

**Pull request merged**